tsconfig.base.json

```
@@ -12,13 +12,17 @@
     "typeRoots": ["node_modules/@types", "./typings"],
     "baseUrl": ".",
     "paths": {
-      "@fluentui/set-version": ["packages/set-version/src/index.ts"],
       "@fluentui/react-conformance": ["packages/react-conformance/src/index.ts"],
       "@fluentui/react-utilities": ["packages/react-utilities/src/index.ts"],
       "@fluentui/react-make-styles": ["packages/react-make-styles/src/index.ts"],
+      "@fluentui/make-styles": ["packages/make-styles/src/index.ts"],
+      "@fluentui/react-provider": ["packages/react-provider/src/index.ts"],
+      "@fluentui/theme": ["packages/theme/src/index.ts"],
+      "@fluentui/react-theme": ["packages/react-theme/src/index.ts"],
+      "@fluentui/react-theme-provider": ["packages/react-theme-provider/src/index.ts"],
       "@fluentui/keyboard-key": ["packages/keyboard-key/src/index.ts"],
-      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"],
-      "@fluentui/react-focus-management": ["packages/react-focus-management/src/index.ts"]
+      "@fluentui/react-focus-management": ["packages/react-focus-management/src/index.ts"],
+      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"]
```

```suggestion
      "@fluentui/react-menu": ["packages/react-menu/src/index.ts"],
      "@fluentui/react-context-selector": ["packages/react-context-selector/src/index.ts"],
```

We should also have `react-context-selector` on this list, https://github.com/microsoft/fluentui/pull/17161#issuecomment-788839593.